homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_zone_control_zone.py

```diff
@@ -251,3 +417,51 @@ async def async_refresh_modes(self) -> None:
         await self.executor.async_execute_command(
             OverkizCommand.REFRESH_TARGET_TEMPERATURE
         )
+
+    @property
+    def min_temp(self) -> float:
+        """Return Minimum Temperature for AC of this group."""
+
+        device_hvac_mode = self.device_hvac_mode
+
+        if device_hvac_mode in (HVACMode.HEAT, HVACMode.HEAT_COOL):
+            return cast(
+                float,
+                self.executor.select_state(
+                    OverkizState.CORE_MINIMUM_HEATING_TARGET_TEMPERATURE
+                ),
+            )
+
+        if device_hvac_mode == HVACMode.COOL:
+            return cast(
+                float,
+                self.executor.select_state(
+                    OverkizState.CORE_MINIMUM_COOLING_TARGET_TEMPERATURE
+                ),
+            )
+
+        return self._attr_min_temp
```

Comment:
Where is this attribute set?

Reply:
Indeed, it was a mistake, I used `super` instead.